### ISSUE TITLE
[Bug] Cant add events to Calendar

### ISSUE THREAD

**Issue by TeddyTeddy** [Etar-Group/Etar-Calendar#1416]
Build version: 1.0.34
Android 13 (GrapheneOS)
Permissions Granted: Calendar, Notifications, Sensors
Permissions Not Allowed: Contacts

Via the widget or Etar app itself, save a calendar event
Expected: Can see the event in its day view. Saving does give no indication of failure.
Observed:  Can't see the event in its day/month/week view

**Comment by marben-olvbar** [comment]
I initially had the same issue, until I realized that there just wasn't any calendar to save events to.

Are you able to select a calendar below the "Event Name" field?
If not, you might have to create an offline calendar in the settings first (or add some other existing calendar).

Maybe this is only a issue on GrapheneOS which doesn't ship a calendar by default and hence has no pre-existing calendar, but I found this behaviour to be pretty confusing at first.

I think this could be improved by showing an error if one tries to create an event without having selected any calendar.
